application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadCommand.java

```diff
@@ -198,6 +200,23 @@ private void closeThread(SlashCommandInteractionEvent event, ThreadChannel helpT
         event.replyEmbeds(embed).flatMap(any -> helpThread.getManager().setArchived(true)).queue();
     }
 
+    private void resetActivity(SlashCommandInteractionEvent event, ThreadChannel helpThread) {
+        refreshCooldownFor(Subcommand.RESET_ACTIVITY, helpThread);
+
+        if (!event.getMember().equals(helpThread.getOwner())) {
+            event.reply("Only the thread creator can reset activities.").queue();
+            return;
+        }
+
+        helpThread.getHistory()
+            .retrievePast(1)
+            .queue(messages -> manuallyResetChannelActivityCache.put(helpThread.getIdLong(),
+                    messages.get(0).getIdLong()));
```

Comment:
add `.map(messages -> messages.get(0))` to then be able to write `.queue(lastMessage -> ...)` enhances readability

Reply:
done.